smartcontractkit/external-adapters-js
# Make Coinpaprika default to the config base URL

## Changes

- Fix issue where the API endpoint could not be changed with the API_ENDPOINT env var.



## Steps to Test

1. Run coinpaprika EA with `API_ENDPOINT` env var set
2. Ensure that the EA uses the `API_ENDPOINT` endpoint

## Quality Assurance
